docs/source/fundamentals/tips.md

```diff
@@ -0,0 +1,48 @@
+
+Luckily, the GraphQL Schema Definition Language (SDL) focuses on the shape of the data and not the actual implementation, allowing product teams to be the ideal custodians of their own GraphQL schemas.
+
+Having product teams own the GraphQL schema for their products allows the schema to act as an API contract within the team and enhances intra-team communication by providing clearly defined expectations.  These teams can utilize schema-driven development which lets them divide back-end and front-end work, mocking various parts of the API until full functionality is achieved, while remaining confident that they will arrive back at a unified solution.
+
+Organizations looking to offer a single API endpoint can assemble the individual product schemas, managed by individual product teams into a monolithic API by “stitching” the various schemas together.
+
+<h2 id="performance">Monitor GraphQL performance</h2>
+
+An API implemented using GraphQL allows developers to query the exact information they desire from an API and nothing more.  With proper visibility into how the API performs, developers can understand the implications of adding or removing fields, especially those which might be slow (notoriously or otherwise!).
+
+Apollo Engine is a GraphQL schema management and monitoring tool which provides insight on how a GraphQL API is behaving, and can proactively notify teams of degraded performance.
+
+When using Apollo Server with Apollo Engine, queries and mutations are enhanced with field-level tracing data which power views in Apollo Engine showing how removing a field from a query might speed up a particular component or, conversely, how adding a seemingly innocent field might actually do more harm than good!
+
+In addition to empowering front-end developers to make quick, educated decisions during development, the detailed metrics in Apollo Engine allow those who are responsible for data performance to quickly identify where improvements can be made and be made aware of bottlenecks or rising error rates if issues arise.
+
+This clarity allows pushing schema changes to production with the confidence of knowing that the API is performing as well as it was before, if not better!
+
+<h2 id="javascript">Write the server in JavaScript</h2>
+
+Facebook's reference implementation has been written in JavaScript since its original release and fresh developments in the GraphQL ecosystem have frequently appeared first in JavaScript or languages which transpile to JavaScript, like TypeScript.
+
+While existing back-end systems might already be implemented in other languages, a thin, JavaScript-based GraphQL implementation fits nicely in front of such systems and, due to the active maintenance of the JavaScript implementations, can bring the latest in GraphQL features to this existing infrastructure.
```

Comment:
We should specifically mention Apollo Server here. Since we're the ones leading innovation in the GraphQL ecosystem with specifications like cache control and tracing, our server implementation is always the most up-to-date with the latest features. We could also mention schema stitching here, since many of the other language servers don't support it yet.

This section needs to answer why JavaScript (which you've done a great job at already) AND why Apollo Server (vs another JS server implementation).

Reply:
I think mentioning Apollo Server here makes sense. However, we can leave out Schema stitching for now.